Ensure correct theme

sawirricardo opened the pull request:
Continued from https://github.com/filamentphp/filament/pull/7217

- [x] Changes have been thoroughly tested to not break existing functionality.
- [x] New functionality has been documented or existing documentation has been updated to reflect changes.
- [x] Visual changes are explained in the PR description using a screenshot/recording of before and after.

sawirricardo committed:
Switch to light theme if ->darkMode(false)

if we disable dark mode, the app should switch to light theme by setting to localStorage

zepfietje commented on packages/panels/resources/views/components/layout/base.blade.php:
```diff
@@ -78,6 +78,16 @@
 
+        @if (! filament()->hasDarkMode())
+            <script>
+                const theme = localStorage.getItem('theme') ?? 'system'
+                
+                if (theme != 'light') {
+                    localStorage.setItem('theme', 'light')
+                }
+            </script>
```
We just need `localStorage.setItem('theme', 'light')`, right?

sawirricardo replied:
That's correct, too

sawirricardo replied:
https://github.com/filamentphp/filament/pull/7224/commits/eabd152d81ea6cf7494326295587662414f5ab45

sawirricardo committed:
set theme to light directly

based on https://github.com/filamentphp/filament/pull/7224#discussion_r1274962257

zepfietje committed:
wip

sawirricardo commented:
nice! thanks @zepfietje
